app/export.go

```diff
@@ -215,6 +218,10 @@ func (a *App) exportAllChannels(writer io.Writer) *model.AppError {
 			if channel.DeleteAt != 0 {
 				continue
 			}
+			// Skip channels on deleted teams.
+			if ok := teamNames[channel.TeamName]; !ok {
```

Comment:
Is this going to work properly with DMs/GMs (as they are teamless)?

Reply:
Hmm, DMs are fine as those are handled separately. I am checking into GMs though as I am not sure how those are handled.

Reply:
Yep, I confirmed that group channels are also handled separately @streamer45